test/onigumo_test.exs

```diff
@@ -39,6 +41,11 @@ defmodule OnigumoTest do
     }
   end
 
+  defp urls_input(urls) do
```

Comment:
```suggestion
  defp prepare_content(urls) do
```

Reply:
Commented in https://github.com/Glutexo/onigumo/pull/44#issuecomment-1029071593.